fw-child/template/query/learn-item.php

```diff
@@ -21,7 +37,7 @@
 		<div class="mb-3">
 			<?php
 			
-				echo apply_filters ( 'the_content', custom_excerpt ( 20, $item['id'] ) );
+				echo $excerpts[ $item['lang'] ];
```

Comment:
As this is an excerpt, it is recommended to add escaping functions like `esc_html()`